app/grants/migrations/0006_auto_20190111_2350.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.1.4 on 2019-01-11 23:50
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('grants', '0005_auto_20190109_2042'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='grant',
+            name='cancel_tx_confirmed',
+            field=models.BooleanField(default=False, help_text='Whether or not the grant cancellation transaction has been confirmed on chain.'),
+        ),
+        migrations.AddField(
+            model_name='grant',
+            name='deploy_tx_confirmed',
```

Comment:
won't this migration make it such that each existing Grant in the DB has `deploy_tx_confirmed` == `False` ?

Reply:
You are right I need to create an empty migrations which update legacy grants